src/win32/posix_w32.c

```diff
@@ -390,12 +394,20 @@ int p_readlink(const char *path, char *buf, size_t bufsiz)
 	return (int)bufsiz;
 }
 
-int p_symlink(const char *old, const char *new)
+int p_symlink(const char *target, const char *path)
 {
-	/* Real symlinks on NTFS require admin privileges. Until this changes,
-	 * libgit2 just creates a text file with the link target in the contents.
-	 */
-	return git_futils_fake_symlink(old, new);
```

Comment:
Do you know off-hand how we treated these fake symlink in our diff machinery? Were those files shown as modified or was there some "magic" around fake symlinks?

Reply:
They weren't shown as different.  The [diff mechanism](https://github.com/libgit2/libgit2/blob/e6cdd17c846648aa8e1e025fa1988475886a551e/src/diff_file.c#L297) will detect if symlinks are supported and either stat the link to determine its destination, or read the file if symlinks are not supported.